wire-library/wire-tests/src/commonTest/proto-kotlin/com/squareup/wire/protos/custom_options/FooBar.kt

```diff
@@ -429,21 +430,26 @@ class FooBar(
     }
   }
 
+  @EnumOption(true)
   enum class FooBarBazEnum(
     override val value: Int,
     val enum_value_option: Int?,
     val complex_enum_value_option: More?,
     val foreign_enum_value_option: Boolean?
```

Comment:
Maybe it's a good time to stop generated those weird constructors for Enums

Reply:
You don't have to ask me twice.